assets/src/components/routeLadder.tsx

```diff
@@ -58,23 +58,40 @@ const Vehicle = ({ vehicle }: { vehicle: Vehicle }) => (
   </ul>
 )
 
-const RouteLadder = ({ route, timepoints, vehicles }: Props) => (
-  <div className="m-route-ladder">
-    <Header route={route} />
+const RouteLadder = ({ route, timepoints, vehicles }: Props) => {
+  const [shouldReverseStops, setShouldReverseStops] = useState(false)
```

Comment:
When we start drawing the vehicles, we'll want a sense of "is direction_id 0 upwards or downwards", and not just "is this the same order or opposite order from the server".

Is that something we can define now? Can we document that the api returns stops in the 0 to 1 order, and then have this be `"0to1" | "1to0"`?

Reply:
Ok, fine with me. I might suggest using something like `ZeroToOne` and `OneToZero` so that they can be keys in an `enum`.

Reply:
Updated